IbisPlugins/GPU_VolumeReconstruction/gpu_volumereconstructionwidget.cpp

```diff
@@ -161,12 +168,13 @@ void GPU_VolumeReconstructionWidget::UpdateUi()
   ui->usAcquisitionComboBox->clear();
   ui->usSearchRadiusComboBox->clear();
   ui->usVolumeSpacingComboBox->clear();
-  SceneManager * sm = m_pluginInterface->GetSceneManager();
-  const SceneManager::ObjectList & allObjects = sm->GetAllObjects();
+  IbisAPI *ibisAPI = m_pluginInterface->GetIbisAPI();
+  Q_ASSERT(ibisAPI);
+  const QList< SceneObject* > allObjects = ibisAPI->GetAllObjects();
```

Comment:
GetAllObjects returns a reference to a list, not the list itself. It will work anyways, but will create a copy of the list instead of using the original as in the old code.